SQL/updates/36-37.py

```diff
@@ -0,0 +1,153 @@
+# :wave: hello fellow contributors, this script is brought to you ad-free by -sirryan2002-
```

Comment:
This needs to be a 3 part update in the same way I did in #17447

`36-37.sql` to rename the `library` table to `library_old`, then make the new tables
`37-38.py` to do the conversion stuff
`38-39.sql` to remove the old table